CSS/Style.css

```diff
@@ -0,0 +1,70 @@
+body{
+    background-color:darkgoldenrod;
+    text-align:center;
+    font-weight: bold;
+}
+[tooltip="Aman"] {
+    position:relative;
+    display:block;
+    padding: 50px;
+    text-align: center;
+    cursor:pointer;
+}
```

Comment:
tooltip="Aman", here we may pass any text values to be displayed on tooltip. You shouldn't consider it as fixed value. Let's say I want my name to be displayed on tooltip, then I will pass tooltip="Ravi" then your complete css style will go on vein.